assets/css/tailwind.css

```diff
@@ -41,7 +41,8 @@
     --color-text-muted: /* #999999 */ 153 153 153;
     --color-text-inverted: /* #ffffff */ 255 255 255;
     --color-bg-primary: /* #FCFCFC */ 252 252 252;
-    --color-bg-secondary: /* #FFF1EF */ 255 241 240;
+    --color-bg-secondary: /* #ffffff */ 255 255 255;
+    --color-bg-tertiary: /* #e5e7eb */ 229 231 235;
```

Comment:
I have add a new bg color, this is for the moderation panel since I notice that for some components we have a gray background color. I'm not sure if this is the best approach, let me know what your opinion is.